src/pipeline_v3/cli/management.py

```
@@ -960,60 +971,66 @@
+        elif args.queue_action == "status":
+            status = self.queue.get_status()
+
             if args.detailed:
                 print(self._format_output(status, args.json))
             else:
                 print(f"Queue: {status.get('state', 'unknown')}")
-                print(f"Jobs: {status.get('pending_jobs', 0)} pending, {status.get('active_jobs', 0)} active")
-                
-        elif args.queue_action == 'clear':
+                print(
+                    f"Jobs: {status.get('pending_jobs', 0)} pending, {status.get('active_jobs', 0)} active"
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)